packages/loader/container-loader/src/container.ts

```diff
@@ -288,7 +288,12 @@ export class Container extends EventEmitterWithErrorHandling<IContainerEvents> i
 
         return PerformanceEvent.timedExecAsync(
             container.mc.logger,
-            { eventName: "Load" },
+            {
+                eventName: "Load",
+                canReconnect: loadOptions?.canReconnect,
```

Comment:
I was considering to stringily and log full loadOptions. But Im assuming the general guideline here is to be very explicit about fields we are collecting telemetry on.

Reply:
This is not needed - see line 318:

 event.end({ ...props, ...loadOptions.loadMode });

If you want to add canReconnect, please add it there. Or if you want to, you can move all up here, but let's not add duplicated code

Reply:
If the data was there and they didn't know about it... this is more reason to move stuff to details field so people can find the info they want!

Reply:
Another idea I've had is to change Bohemia to include a `Data_telemetryKeys` field which lists all the properties set on that event.  Thoughts?

Reply:
To clarify what I said aboe -- My suggestion is to put anything from loadOptions into a JSON payload and stringify it into a field called "details".  This is already done in a number of places, so including Data_details in queries will show you extra info applicable to each event.  If you need to aggregate/filter on that, Kusto lets you materialize columns from a JSON-value'd column.

As for this:

> I was considering to stringily and log full loadOptions

Looking at `IContainerLoadOptions` you're right to hesitate on that, because it has an `IFluidResolvedUrl` on it which includes URLs and tokens and stuff!

Reply:
So it was on the `event.end` then. When I looked at the logs during oce shift, I assumed that `start` collects any telemetry that's relevant/known at the start of the event, while `end` collects telemetry that was the outcome (known only at the end) of the call we are tracking. I moved these items to `start` and wrapped them in "details"

Loading this data in `details` is another explicit way of stating that this (additional) data is tightly coupled with one particular event. And that does help discoverability.